core/src/main/resources/hudson/model/Job/configure_ja.properties

```diff
@@ -1,6 +1,6 @@
 # The MIT License
 # 
-# Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi, id:cactusman
+# Copyright (c) 2004-2023, Sun Microsystems, Inc., Kohsuke Kawaguchi, id:cactusman, Takashi Harano
```

Comment:
Since Sun Microsystems is no longer a legal entity, I would recommend leaving this line as-is and adding a new line beneath it to add your own 2023 copyright instead.

Reply:
@basil 
Thanks for your comment.
I modified the copyright statement as you pointed out.